reference/performances_guide/index.rst

```diff
@@ -3,7 +3,7 @@ Performances Guide
 
 Your product catalog is unique, with different amount of attributes, families, locales, channels, products, etc all those different combinations may cause different performance issues.
 
-The following chapter explains the different performances bottlenecks you can encounter by using Akeneo PIM and list known issues and related best practices you should implement to ensure the success of your project.
+The following chapter explains the different performances bottlenecks you can encounter by using *Akeneo PIM* and list known issues and related best practices you should implement to ensure the success of your project.
```

Comment:
[...] and lists [...]

Your sentnence is too long, you should split it in two parts.  I think you can have:
[...] known issues. It also explains best practices [...]